src/common/util/feature-flags.js

```diff
@@ -15,12 +16,6 @@ export function activateFeatures (flags, agentIdentifier) {
     sharedEE.emit('feat-' + flag, [])
     activatedFeatures[flag] = true
   })
-  drainAll(agentIdentifier)
```

Comment:
activateFeatures (result of RUM call) is no longer responsible for draining.  Each feature buffers to a cache with the feature's name.  and each feature's agg file is responsible for draining its own cache

Reply:
For commenting later, one here also to point out how PVE drain is separate from its agg module and why - thanks.

Reply:
[_https://issues.newrelic.com/browse/NEWRELIC-6479_](https://issues.newrelic.com/browse/NEWRELIC-6479)